scripts/mesh_visualizer.py

```diff
@@ -32,37 +27,53 @@ def visualize(env_file, result_file):
     with open(env_file) as f:
         data = yaml.load(f, Loader=yaml.FullLoader)
         
+    obstacles = data["environment"]["obstacles"]
+    for k, obs in enumerate(obstacles):
+      center = obs["center"]
+      size = obs["size"]
+      obs_type = obs["type"]
+      if (obs_type == 'octomap'):
+         vis[f"Obstacle{k}"].set_object(g.StlMeshGeometry.from_file('../meshes/map.stl'), g.MeshLambertMaterial(color="blue")) # hard-coded
```

Comment:
this should come from the yaml file: if there is an octomap, there should be a filename